app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -74,27 +74,33 @@ def configuration_script_sources
     end
   end
 
+  def miq_credential_type(tower_type)
```

Comment:
Recommend `tower_credential_type` to be consistent with the method name.

Reply:
thanks! Done